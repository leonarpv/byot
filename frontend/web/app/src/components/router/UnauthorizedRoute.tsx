import * as React from 'react';
import {RouteProps} from 'react-router-dom';
import {UnauthorizedRouteBase} from '@byot-frontend/web-common/src/components/router/UnauthorizedRouteBase';
import {Router} from '../../router/Router';

interface Props extends RouteProps {}

export const UnauthorizedRoute: React.FC<Props> = (props: Props) => {
  return <UnauthorizedRouteBase authorizedUrl={Router.URI()} {...props} />;
};
