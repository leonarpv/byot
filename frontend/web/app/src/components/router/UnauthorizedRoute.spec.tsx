import * as React from 'react';
import {render} from '@testing-library/react';
import {UnauthorizedRoute} from './UnauthorizedRoute';

describe('<UnauthorizedRoute/>', () => {
  it('should render', () => {
    const {container} = render(<UnauthorizedRoute />);
    expect(container).toMatchSnapshot();
  });
});
