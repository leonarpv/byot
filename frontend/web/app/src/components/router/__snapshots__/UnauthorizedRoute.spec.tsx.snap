// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UnauthorizedRoute/> should render 1`] = `
<div>
  <div
    data-testid="unauthorized-route-base"
  />
</div>
`;
